server/config/config.go

```diff
@@ -966,16 +972,30 @@ func (c *ReplicationConfig) clone() *ReplicationConfig {
 		LocationLabels:       locationLabels,
 		StrictlyMatchLabel:   c.StrictlyMatchLabel,
 		EnablePlacementRules: c.EnablePlacementRules,
+		IsolationLabel:       c.IsolationLabel,
 	}
 }
 
 // Validate is used to validate if some replication configurations are right.
 func (c *ReplicationConfig) Validate() error {
+	validIsolationLabel := false
 	for _, label := range c.LocationLabels {
 		err := ValidateLabels([]*metapb.StoreLabel{{Key: label}})
 		if err != nil {
 			return err
 		}
+		if !validIsolationLabel && label == c.IsolationLabel {
+			validIsolationLabel = true
+		}
+	}
+	if len(c.IsolationLabel) > 0 {
+		if !validIsolationLabel {
+			return errors.New("invalid isolation-label")
+		}
+		err := ValidateLabels([]*metapb.StoreLabel{{Key: c.IsolationLabel}})
```

Comment:
Do we need this? Maybe it has already validated by the above loop? Another question is what if we don't set the LocationLabels and only set IsolationLabel? It seems that it will return an error. Does it meet your expectation?

Reply:
You're right. Above loop guarantees this.